src/frequenz/sdk/timeseries/_formula_engine/_formula_generators/_pv_power_formula.py

```diff
@@ -66,3 +67,39 @@ def generate(self) -> FormulaEngine:
             builder.push_clipper(0.0, None)
 
         return builder.build()
+
+    def _get_pv_meters(self) -> abc.Set[int] | None:
```

Comment:
Documentation is missing here.

Reply:
It is a private function.  we don't need to document them unless they are not obvious.

Reply:
ok, I didn't know we can omit documentation for trivial internal methods